src/VirtoCommerce.ElasticAppSearch.Core/Models/Api/Json/SearchModuleCoreGeoPointConverter.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Newtonsoft.Json;
+using SearchModuleCoreGeoPoint = VirtoCommerce.SearchModule.Core.Model.GeoPoint;
+
+namespace VirtoCommerce.ElasticAppSearch.Core.Models.Api.Json
+{
+    public class SearchModuleCoreGeoPointConverter : JsonConverter<SearchModuleCoreGeoPoint>
```

Comment:
Why does we need this one? I though we always convert GeoPoint from Search module to API model